bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx117-vpn.html

```diff
@@ -0,0 +1,211 @@
+  {% set vpn_cta = "Ia-ți Mozilla VPN" %}
+  {% set vpn_signoff = "<strong>Powered by Mozilla.</strong> Putting people before profits since 1998."|safe %}
+
+{%- elif LANG == "ru": -%}
+  {% set vpn_title = "Конфиденциальность следующего уровня будет вашей в один клик" %}
+  {% set vpn_tagline = "Лето, может быть, и подходит к концу, но не ваша безопасность.Mozilla VPN — это ваше решение одним кликом для дополнительного душевного спокойствия в течение всего года." %}
+  {% set vpn_cta = "Установить Mozilla VPN" %}
+  {% set vpn_signoff = "<strong>На базе Mozilla.</strong> Ставим людей выше прибыли с 1998 года."|safe %}
+
+{%- elif LANG == "sk": -%}
+  {% set vpn_title = "Súkromie na vyššej úrovni je vzdialené len jedno kliknutie." %}
+  {% set vpn_tagline = "Leto sa síce končí, ale vaša bezpečnosť by sa nemala.VPN Mozilla je vaše riešenie na jedno kliknutie pre špeciálny pocit istoty počas celého roka." %}
+  {% set vpn_cta = "Získajte VPN Mozilla" %}
+  {% set vpn_signoff = "<strong>Podporované spoločnosťou Mozilla.</strong> Ľudí uprednostňujeme pred ziskom už od roku 1998."|safe %}
+
+{%- elif LANG == "sl": -%}
+  {% set vpn_title = "Od napredne stopnje zasebnosti vas loči zgolj en klik." %}
+  {% set vpn_tagline = "Poletje se poslavlja, vaša varnost pa ne. Z rešitvijo Mozilla VPN boste brez skrbi vse leto." %}
+  {% set vpn_cta = "Prenesite Mozilla VPN" %}
+  {% set vpn_signoff = "<strong>Rešitev omogoča Mozilla.</strong> Ljudi postavljamo pred dobiček že od leta 1998."|safe %}
+
+{%- elif LANG == "sv-SE": -%}
+  {% set vpn_title = "Ta sekretessen till nästa nivå med bara ett klick" %}
+  {% set vpn_tagline = "Även om sommaren varvar ner, bör inte din säkerhet göra det. Mozilla VPN ger dig sinnesro hela året med bara ett klick." %}
+  {% set vpn_cta = "Skaffa Mozilla VPN" %}
+  {% set vpn_signoff = "<strong>Drivs av Mozilla.</strong> Värderar människor mer än vinst sedan 1998."|safe %}
+
+{%- else: -%}
+  {% set vpn_title = "Next-level privacy is just a click away" %}
+  {% set vpn_tagline = "Summer may be winding down, but your security shouldn’t. Mozilla VPN is your one-click solution for extra peace of mind all year long." %}
+  {% set vpn_cta = "Get Mozilla VPN" %}
+  {% set vpn_signoff = "<strong>Powered by Mozilla.</strong> Putting people before profits since 1998."|safe %}
+{%- endif -%}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+        <div class="wnp-main-image">
+          <span class="toggle" role="presentation"><span class="toggle-display"></span></span>
+        </div>
+
+        <h2 class="wnp-main-title">{{ vpn_title }}</h2>
+
+        <p class="wnp-main-tagline">
+          {{ vpn_tagline }}
+        </p>
+
+        <p class="wnp-main-cta">
+        {% if variant == "2" %}
+          {{ vpn_product_referral_link(
+            page_anchor='#pricing',
+            referral_id='whatsnew-117-eu-vpn',
```

Comment:
Suggest making `referral_id` slightly different so we can tell which variation performed better?
```suggestion
            referral_id='whatsnew-117-eu-vpn-pricing',
```

Reply:
doh! Good catch